src/util/constants.ts

```diff
@@ -10,5 +10,5 @@ export enum Path {
   PROBLEMS = "/problems",
   CONTESTS = "/contests",
   Stats = "/stats",
-  Issues = "/comments"
+  Issues = "/issues"
```

Comment:
Sorry. After thinking a lot, I think it would be better to rename Issues in Menu to Comments, as I would like people to share suggestions or any features they want besides issues there.

Reply:
okay got it, if that was the aim then it better be /comments ...